src/MCPClient/lib/clientScripts/dip_generation_helper.py

```diff
@@ -36,9 +39,11 @@ def create_archivesspace_client():
             repository=config['%repository%']
         )
     except archivesspace.AuthenticationError:
+        logging.error('Unable to authenticate to ArchivesSpace server using the default user! Check administrative settings.')
```

Comment:
These should be `logger` not `logging`.  `logging` logs to the root logger, while `logger` logs to the define logger (below), and it's very useful for determining where a log originated and getting it filtered to the correct log files to use defined loggers.